react/src/pages/participants/SampleTable.tsx

```diff
@@ -1,54 +1,116 @@
 import React from "react";
 import { makeStyles } from "@material-ui/core";
-import MaterialTable from "material-table";
-import { formatDateString, Sample } from "../utils";
-import DatasetAccordions from "./DatasetAccordion";
+import { Dns } from "@material-ui/icons";
+import MaterialTable, { MTableCell } from "material-table";
+import { formatDateString } from "../utils/functions";
+import { Dataset, Sample, Info } from "../utils/typings";
+import InfoList from "../utils/components/InfoList";
 
 const useStyles = makeStyles(theme => ({
-    table: {
-        marginBottom: theme.spacing(3),
+    datasetList: {
+        marginLeft: theme.spacing(3),
+        marginRight: theme.spacing(3),
     },
 }));
 
-interface SamplesTableProp {
-    samples: Sample[];
+function getTitles() {
+    return [
+        "Input HPF Path",
+        "Condition",
+        "Extraction Protocol",
+        "Capture Kit",
+        "Library Prep Method",
+        "Library Prep Date",
+        "Read Length",
+        "Read Type",
+        "Sequencing ID",
+        "Sequencing Centre",
+        "Creation Time",
+        "Created By",
+        "Last Updated",
+        "Updated By",
+        "Discriminator",
+    ];
+}
+function getValues(dataset: Dataset) {
+    return [
+        dataset.input_hpf_path,
+        dataset.condition,
+        dataset.extraction_protocol,
+        dataset.capture_kit,
+        dataset.library_prep_method,
+        formatDateString(dataset.library_prep_date),
+        dataset.read_length,
+        dataset.read_type,
+        dataset.sequencing_id,
+        dataset.sequencing_centre,
+        formatDateString(dataset.created),
+        dataset.created_by,
+        formatDateString(dataset.updated),
+        dataset.updated_by,
+        dataset.discriminator,
+    ];
 }
 
-export default function SamplesTable({ samples }: SamplesTableProp) {
+export default function SamplesTable({ samples }: { samples: Sample[] }) {
```

Comment:
```suggestion
export default function SampleTable({ samples }: { samples: Sample[] }) {
```
This is always referred to as this everywhere else.